modules/installation-user-infra-machines-advanced-customizing-live-serial-console.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+
+:_content-type: PROCEDURE
+[id="installation-user-infra-machines-advanced-customizing-live-{boot}-serial-console_{context}"]
+= Modifying a live install {boot-media} to enable the serial console
+
+By default, the serial console is disabled and all output is written to the graphical console. You can enable the serial console with the following procedure.
```

Comment:
This procedure also disables the graphical console, which may not be desired.

In general, this text assumes that the user understands what they need to do, and just focuses on the how.  We should probably start by mentioning some install scenarios which benefit from a serial console (a bare metal SME could help here) and which debugging scenarios (loss of network connectivity, Ignition failures, boot failures) require access to a working console.  We might also want to explain that they can have one console of each type, that the primary console is the last one specified, and that the secondary console receives fewer types of logging information.

Also, should we say "On clusters installed with 4.12 and above, the serial console is disabled by default..."?

Reply:
> In general, this text assumes that the user understands what they need to do, and just focuses on the how

This is just how it goes for procedure modules, but I can add a concept module that goes more in-depth on these topics. 

> Also, should we say "On clusters installed with 4.12 and above, the serial console is disabled by default..."?

Probably, just to avoid any confusion wrt upgrading existing clusters 🙂